checkov/gitlab_ci/runner.py

```diff
@@ -91,3 +101,15 @@ def extract_images(
             images.extend(manager.extract_images_from_workflow())
 
         return images
+
+    @staticmethod
+    def get_start_and_end_lines(key: str) -> list[int]:
+        check_name = key.split('.')[-1]
+        try:
+            start_end_line_bracket_index = check_name.index('[')
+            closing_bracket_index = check_name.index(']')
+            if closing_bracket_index - start_end_line_bracket_index == 1:
```

Comment:
```suggestion
            if "[" not in check_name or "[]" in check_name:
```
how about this? then you also don't need the `try/except` block